hr_course/migrations/13.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,33 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+_models_renames = [
+    ("hr.course", "hr.course.schedule"),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_models(env, _models_renames)
+
+    openupgrade.logged_query(
+        env.cr,
+        """
+        SELECT id, name, category_id, permanence, permanence_time
+        INTO hr_course
```

Comment:
I would prefer to rewrite id in order to be sure that no we have no problem with the identifier sequence. On the other hand, this kinds of inserts are usually done on post-migration.

When you want to manage this kind of problems, you should need to add an old_course_id (or something like that) on hr.course.schedule in order to make it work (similar to account.invoice to account.move migration from 12 to 13)

Reply:
Can you review the solution?